COMPILING.md

```diff
@@ -114,6 +114,25 @@ files.
    complete set of built binaries can be found in `./build/bin` once the build
    is complete.
 
+## Compilation on Windows using Visual Studio
+
+Follow these instructions to work on Windows with Visual Studio:
+
+1. First, clone the `cbmc` repository locally.
```

Comment:
Fair call, but other build instructions don't suggest that. We should be consistent about that.